cranelift/interpreter/src/value.rs

```diff
@@ -277,6 +278,16 @@ impl Value for DataValue {
         Ok(self.is_nan()? || other.is_nan()?)
     }
 
+    fn of(&self, other: &Self) -> ValueResult<bool> {
+        Ok(match (self, other) {
+            (DataValue::I8(a), DataValue::I8(b)) => a.checked_sub(*b).is_none(),
+            (DataValue::I16(a), DataValue::I16(b)) => a.checked_sub(*b).is_none(),
+            (DataValue::I32(a), DataValue::I32(b)) => a.checked_sub(*b).is_none(),
+            (DataValue::I64(a), DataValue::I64(b)) => a.checked_sub(*b).is_none(),
+            _ => unimplemented!(),
+        })
+    }
```

Comment:
Ah, I see... I thought this meant "of" as in "a value of..." but I think it is an abbreviation of "overflows," right? Let's just go with the longer version to avoid confusion.